src/Glpi/Form/Condition/ConditionData.php

```diff
@@ -84,6 +86,16 @@ public function getValueOperator(): ?ValueOperator
         return ValueOperator::tryFrom($this->value_operator ?? "");
     }
 
+    /**
+     * Compute the UUID of the condition.
+     *
+     * @return string
+     */
+    public function getUuid(): string
+    {
+        return md5(json_encode($this->jsonSerialize()));
```

Comment:
Does not seems like a UUID since it will return the same results for two identical conditions.